src/inc/quic_datapath.h

```diff
@@ -444,6 +455,7 @@ CxPlatDataPathUpdateConfig(
 #define CXPLAT_DATAPATH_FEATURE_TCP                   0x0020
 #define CXPLAT_DATAPATH_FEATURE_RAW                   0x0040
 #define CXPLAT_DATAPATH_FEATURE_TTL                   0x0080
+#define CXPLAT_DATAPATH_FEATURE_DSCP                  0x0100
```

Comment:
Will we need separate features for send/receive support?

Reply:
I *think* the receive support already exists in all stacks, it's just the send support that's missing on some.

Reply:
Given that receive support seems to not be working on Windows in CI, I am going to make separate features for send and receive.